### DISCUSSION TITLE
Can't hear race assistants in Accetta Corsa

### DISCUSSION THREAD

**Discussion by stuboyle** [SeriousOldMan/Simulator-Controller#15] [Q&A] [answered]
Thanks for the great app!  Now, I've got AC, AM2 and RF2.  The race assistants are audible in AM2 and RF2 but not AC.  Actually, I'm not sure to what extent any of your app is working in AC.  I'm just try to get the race assistants to work as a start.  Now in AC, I normally launch "content manager.exe" which I believes calls on  "assettacorsa.exe".   I did try launching assettacorsa.exe directly but still didn't hear the race assistants.  Everything is installed in the default location.  Any suggestions?

Thanks,

Stuart

**Comment by SeriousOldMan** [comment, marked as answer]
Hi Stuart,

this looks like the Assistants does not startup in AC at all. Have you installed the data acquisition plugin for AC as described in the documentation for the "AC" plugin: 

https://github.com/SeriousOldMan/Simulator-Controller/wiki/Virtual-Race-Engineer#installation-of-telemetry-providers

-------------------------------------------------

Installation of Telemetry Providers

Jona acquires telemetry data from the different simulation games using so called telemtry providers, which in most cases read the [required data](https://github.com/SeriousOldMan/Simulator-Controller/wiki/Virtual-Race-Engineer#telemetry-integration) from a shared memory interface. In general these are already included in Simulator Controller and there is nothing to do, but for Assetto Corsa and rFactor 2, you need to install a plugin into a special location for the telemetry interface to work and for Automobilista 2 and Project CARS 2 a change in the settings is necessary.

Please, note that all of this also applices to all other Virtual Race Assistants, but you have to do it only once, of 

course.

Assetto Corsa

Please copy the complete SimlatorController folder, which is located in the Utilities\Plugins folder of the Simulator Controller installation, to the Steam installation folder of Assetto Corsa and there into the apps\python folder. You will have to enable this plugin in the Asseto Corsa settings afterwards. This plugin uses code originally developed by Sparten which can be found at [GitHub](https://github.com/Sparten/ACInternalMemoryReader).

rFactor 2

You can find the plugin rFactor2SharedMemoryMapPlugin64.dll in Utilities\Plugins folder the Simulator Controller installation or you can load the [latest version](https://github.com/TheIronWolfModding/rF2SharedMemoryMapPlugin) from GitHub. Copy the DLL file to the Bin64\Plugins folder in the Steam installation directory of rFactor 2.

Automobilista 2 and Project CARS 2

You have to enable Shared Memory access in the game settings. Please use the PCars 2 mode.

**Comment by stuboyle** [comment]
Thanks!  I got it working.  Following your direction for copying the folder for AC did the trick.  For RF2, the DLL was already present and for AMS2 the shared memory access had already been enable.  This explains why they worked.
